src/software/embedded/services/motor.h

```diff
@@ -403,24 +442,19 @@ class MotorService
     int back_left_target_rpm   = 0;
     int back_right_target_rpm  = 0;
 
-    // the motor cs id to check for motor faults
-    uint8_t motor_fault_detector_;
```

Comment:
as per our discussion, this was intentional: https://github.com/UBC-Thunderbots/Software/pull/3425/files#r2217047185